src/views/schema/upstream/edit.vue

```diff
@@ -155,6 +167,49 @@ export default class extends Vue {
 
   private types = ['roundrobin', 'chash']
 
+  private defaultHashKeys = [
+    {
+      value: 'remote_addr',
+      label: 'remote_addr'
+    },
+    {
+      value: 'host',
+      label: 'host'
+    },
+    {
+      value: 'uri',
+      label: 'uri'
+    },
+    {
+      value: 'server_name',
+      label: 'server_name'
+    },
+    {
+      value: 'server_addr',
+      label: 'server_addr'
+    },
+    {
+      value: 'request_uri',
+      label: 'request_uri'
+    },
+    {
+      value: 'query_string',
+      label: 'query_string'
+    },
+    {
+      value: 'remote_port',
+      label: 'remote_port'
+    },
+    {
+      value: 'hostname',
+      label: 'hostname'
+    },
+    {
+      value: 'arg_id',
```

Comment:
`arg_id` is wrong, here is the Nginx variable index: http://nginx.org/en/docs/http/ngx_http_core_module.html#var_arg_

`arg_name`: argument name in the request line, the value of the name here is not fixed, eg: `arg_id`, `arg_name`, `arg_age`.

Reply:
I put the arg_id here just reminder  that arg_*** is able to input here

Reply:
It looks everything here is ok, I will approve soon.